packages/mdc-floating-label/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@material/floating-label",
+  "description": "The Material Components for the web floating-label component",
+  "version": "0.30.0",
```

Comment:
`0.31.0` now 😄

Reply:
New packages and dependencies on them should be initialized as 0.0.0 and lerna will rev them during the next release.

See e.g. https://github.com/material-components/material-components-web/commit/1dc0e857b7f117968c99cd46f41f4ae436557fed#diff-2aab7ec5d1d3bc64e9cd3cd98216fa15R4 from when line-ripple was added.

Reply:
Thanks! I'll update